src/pacer.js

```diff
@@ -407,11 +407,17 @@ let PACER = {
     // this regex will match the doc_id and case_id passed as an argument in the
     // onclick event of each anchor element related to a document from Appellate Pacer.
     let doDocPost = /^return doDocPostURL\('([^']*)','([^']*)'\);/.exec(doDoc_string);
-    if (!doDocPost) {
+    let doDocPostAttachment = /^return doDocPostURL\('([^']*)'\)/.exec(doDoc_string);
```

Comment:
An example of this would be nice in the docstring, please.

Reply:
You're right! I will add an example. Thanks.